### PR TITLE
crud_nha_xuat_ban

### PR DESCRIPTION
**Pull request by VanHoa18042000**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51517)

## WHAT (optional)
- Add theme and handle for publisher

## HOW
- Add css, js, view, controller for publisher page 

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

Screenshot from 2022-08-12 07-29-01

### CODE REVIEW

**Comment by VanHoa18042000** [comment]
ready

File: app/assets/stylesheets/admin/publishers.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the admin/Publishers controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
Comment (anlvq-1058) [inline app/assets/stylesheets/admin/publishers.scss:3]: Xóa đi em

File: app/controllers/admin/publishers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,86 @@
+class Admin::PublishersController < ApplicationController
+  layout "admin"
+  # before_action :logged_in_user
```
Comment (anlvq-1058) [inline app/controllers/admin/publishers_controller.rb:3]: ko dùng thì xóa di em

File: app/controllers/admin/publishers_controller.rb [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  before_action :find_by_id, only: %i(show destroy edit update)
```
Comment (anlvq-1058) [inline app/controllers/admin/publishers_controller.rb:4]: Dùng except

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by VanHoa18042000** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**